packages/shared/src/shared/Formatting.ts

```diff
@@ -37,6 +37,7 @@ export function createDefaultValueFormatter(): IPrimitiveValueFormatter {
         return result;
       }
     }
+    // eslint-disable-next-line @typescript-eslint/no-base-to-string
```

Comment:
would this help avoid ignoring the rule?

```suggestion
    assert(typeof x === "undefined" || typeof x === "boolean" || typeof x === "number" || typeof x === "string");
```

Reply:
Yes, added assert